namespaces/live.cloud-platform.service.justice.gov.uk/laa-assure-hmrc-data-staging/certificate.yaml

```
@@ -10,3 +10,16 @@ spec:
     kind: ClusterIssuer
   dnsNames:
   - staging.assure-hmrc-data.service.justice.gov.uk
+---
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: check-clients-details-using-hmrc-data-certificate
+  namespace: laa-assure-hmrc-data-staging
+spec:
+  secretName: check-clients-details-using-hmrc-data-tls-certificate
+  issuerRef:
+    name: letsencrypt-production
+    kind: ClusterIssuer
+  dnsNames:
+  - staging.check-clients-details-using-hmrc-data.service.justice.gov.uk
```

@kmahern  I talked to Vijay. This, below, will do what we want.

```suggestion
  - stg.check-clients-details-using-hmrc-data.service.justice.gov.uk
  - staging.check-clients-details-using-hmrc-data.service.justice.gov.uk
```